google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java

```diff
@@ -332,8 +317,8 @@ public boolean equals(Object o) {
   }
 
   static final class FieldOrder implements Comparator<QueryDocumentSnapshot> {
-    private final FieldReference fieldReference;
-    private final Direction direction;
+    final FieldReference fieldReference;
+    final Direction direction;
 
```

Comment:
Why are these no longer private?

Reply:
Artifact of early approach to refactoring. Thanks!